wordpress-mobile/WordPress-iOS
# Giphy: Import selected media from picker

Fixes #10089. This PR adds support for importing gifs from Giphy that have been selected in the Giphy picker. I made use of the existing `AnimatedImageCache` to fetch the full size image.

![gif-import-1](https://user-images.githubusercontent.com/4780/45208084-a7c48680-b281-11e8-8ba1-0e96a2ad30eb.gif)

**To test:**

* Build and run
* Navigate to the media library for a site
* Tap + and choose Giphy
* Try a search and attempt to add some images to your library
* Check they're imported correctly
